crates/anvil-polkadot/tests/it/snapshot.rs

```diff
@@ -528,6 +537,8 @@ async fn test_mine_with_txs_in_mempool_before_revert() {
         .unwrap(),
     )
     .unwrap();
+    // Previous txs are not included in the block because they have
+    // a gas_price
```

Comment:
```suggestion
    // Previous txs are not included in the block because they have
    // a gas_price smaller than the current block's base_fee_per_gas.
```